near-contract-standards/src/fungible_token/core_impl.rs

```diff
@@ -28,10 +28,10 @@ const ERR_TOTAL_SUPPLY_OVERFLOW: &str = "Total supply overflow";
 #[borsh(crate = "near_sdk::borsh")]
 pub struct FungibleToken {
     /// AccountID -> Account balance.
-    pub accounts: LookupMap<AccountId, Balance>,
+    pub accounts: LookupMap<AccountId, NearToken>,
```

Comment:
Move Balance type alias to be part of FungibleToken module. It is not NearToken, and it was an abuse of Balance type in the first place.